docs/user/datastores/authorizations.rst

```diff
@@ -50,19 +50,19 @@ provider class to use can be specified by the following system property:
     org.locationtech.geomesa.security.AuthorizationsProvider.AUTH_PROVIDER_SYS_PROPERTY
 
 For simple scenarios, the set of authorizations to apply to all queries can be specified when creating
-the GeoMesa data store by using the ``auths`` configuration parameter. This will use a
+the GeoMesa data store by using the ``geomesa.security.auths`` configuration parameter. This will use a
```

Comment:
Which stores support this security configuration?  (And is there a good way to document it?)

Reply:
they're documented in the individual data store parameter sections.

Reply:
Works for me.

Reply:
at the top and bottom of this section it says 'accumulo and hbase datastores' - technically this will work with kafka too. we might want to reorganize this part whenever we get around to revamping the docs